### ISSUE TITLE
Allow ignoring constructors

### ISSUE THREAD

**Issue by cmeeren** [ulrichb/XmlDocInspections#8]
I'm normally using DI/IoC in my apps. In other words, the IoC container instantiates most of my objects and thus very few of my constructors are ever called "manually". For example:

```c#
/// <summary>
///     Provides the ability to flush floobargles into a sink.
/// </summary>
public class SinkFlusher : IFlooFlusher
{
    private readonly ISinkProvider _sinkProvider;

    public IfcModelOperations(ISinkProvider sinkProvider)
    {
        _sinkProvider = sinkProvider;
    }

    /// <summary>
    ///     Flushes the floobargle <see cref="floo" />.
    /// </summary>
    public void Flush(Floobargle floo)
    {
        _sinkprovider.GetSink().Flush(floo);
    }

}
```

Even for the few manual constructors I call, most would only have a pointless "Initializes a new instance of the X class" description, which I'd rather be without. For example:

```c#
/// <summary>
///     Floobargles are used for enhancing the app's flushing experience by around 37%.
/// </summary>
public class Floobargle
{
    public Floobargle(string name)
    {
        Name = name;
    }

    public string Name { get; }

}
```

A constructor doc comment would be pointless here.

In summary, I would like an option for Xml Doc Inspections to ignore constructors.

If not for me, then please do it for the floobargles. The poor guys...

**Comment by ulrichb** [comment]
Implemented on [dev](https://github.com/ulrichb/XmlDocInspections/tree/dev) branch. Will be available in the next release together with R# 2017.1 support.

**Comment by cmeeren** [comment]
Great, thank! :)

**Comment by ulrichb** [comment]
R# 2017.1 is already in the EAP phase. So maybe next month?

**Comment by matkoch** [comment]
@cmeeren what he's trying to ask, would next month be okay for you?

**Comment by cmeeren** [comment]
Oh, didn't know it was a question. Sure, that's fine - and I'm in no position to make any demands anyway. ;)

**Comment by ulrichb** [comment]
Released with 1.8.0.
